telegram/commands/gemini/gemini.go

```diff
@@ -0,0 +1,27 @@
+package gemini
+
+import (
+	"activity-tracker/gemini"
+	"activity-tracker/shared"
+	"errors"
+	"fmt"
+)
+
+var (
+	errNoPrompt = errors.New("En serio metiste un prompt vacío bobo hijueputa? Haz algo bien sub-humano 🙄")
+	ForceText   = "responde solo texto"
```

Comment:
why is this var exportable?

Reply:
look dicksucker, u just made a comment above and i quote 'additionally, set the string as a global var' and i did this, if this is not how u do it then i dont wtf to do

Reply:
you are using unnecessary exportable declaration, why do u create something that u won't use?

Reply:
solved, now stop pissing me off or i will touch myself with your profile pic